service/src/main/resources/db/changesets/20230530_avoid_postgres_specific_features.yaml

```diff
@@ -104,17 +120,17 @@ databaseChangeLog:
             columns:
               - column:
                   name: enum
-                  type: text
+                  type: ${_text.type}
```

Comment:
Same as above. This column is part of the unique constraint, should it be `id.type`?

Reply:
yes, if it is part of unique constraint, varchar(255) is OK too if key length if >50.